### PR TITLE
Makartkar/torch

### PR DESCRIPTION
**Pull request by MakArtKar**
**PyTorchInroduction module**

1) `WorkingWithTensors` -  7 easy tasks to understand simple `torch.Tensor` operations
2) `Autograd` - working with torch automatic gradient calculation
3) `NeuralNetwork` - building simple net
4) `WordWindowDataset` - working with `torch.utils.data.Dataset` and `torch.utils.data.DataLoader`. Write custom `collate_fn` function for `DataLoader`
5) `WordWindowClassifier` - building a network for WordWindow task
6) `WordWindowTraining` - combining 2 previous tasks build a torch pipeline for WordWindow task. Working with `torch.optim.Optimizer` and `torch.nn.BCELoss`

### CODE REVIEW

**Comment by avalur** [comment]
@MakArtKar, thanks a lot, the new assignment is impressive!
